datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedPartitionAssigner.java

```diff
@@ -80,10 +80,12 @@ public LoadBasedPartitionAssigner(int defaultPartitionBytesInKBRate, int default
   public Map<String, Set<DatastreamTask>> assignPartitions(
       ClusterThroughputInfo throughputInfo, Map<String, Set<DatastreamTask>> currentAssignment,
       List<String> unassignedPartitions, DatastreamGroupPartitionsMetadata partitionMetadata, int maxPartitionsPerTask) {
+    LOG.info("START: assignPartitions");
```

Comment:
do we need a datastream name here?

Reply:
Added the data stream name to the START and END markers. This will be sufficient to trace these in the logs, since coordinator is running in a single thread and invocations of this method won't run concurrently.